quickstart/java-redisson/README.md

```diff
@@ -0,0 +1,54 @@
+---
+page_type: sample
+languages:
+- java
+name: 'Quickstart: Use Azure Cache for Redis in Java with Redisson Redis client'
+description: Learn how to incorporate Azure Cache for Redis into a Java app using the Redisson Redis client.
+products:
+- azure
+- azure-cache-redis
+---
+# Quickstart: Use Azure Cache for Redis in Java with Redisson Redis client
+
+This sample show you how to incorporate Azure Cache for Redis into a Java app using the [Redisson](https://redisson.org/) Redis client. See the [accompanying article](https://aka.ms/azure-cache-java-redisson-quickstart) on the documentation site for details, including best practices and how to create the sample code from scratch.
+
+## Prerequisites
+
+- Azure subscription - [create one for free](https://azure.microsoft.com/free/)
+- Azure Cache for Redis cache - [create one](https://docs.microsoft.com/azure/azure-cache-for-redis/quickstart-create-redis)
+- [Apache Maven](https://maven.apache.org/download.cgi)
+
+## Set up the working environment
+
+Depending on your operating system, add environment variables for your cache's **Host name** and **Primary access key**. Open a command prompt, or a terminal window, and set up the following values:
+
+```CMD
+set REDISCACHEHOSTNAME=<YOUR_HOST_NAME>.redis.cache.windows.net
+set REDISCACHEKEY=<YOUR_PRIMARY_ACCESS_KEY>
+```
+
+```bash
+export REDISCACHEHOSTNAME=<YOUR_HOST_NAME>.redis.cache.windows.net
+export REDISCACHEKEY=<YOUR_PRIMARY_ACCESS_KEY>
+```
+
+Replace the placeholders with the following values:
+
+- `<YOUR_HOST_NAME>`: The DNS host name, obtained from the *Properties* section of your Azure Cache for Redis resource in the Azure portal.
+- `<YOUR_PRIMARY_ACCESS_KEY>`: The primary access key, obtained from the *Access keys* section of your Azure Cache for Redis resource in the Azure portal.
+
+## Run the sample
+
+[Download the sample code to your development PC.](/README.md#get-the-samples)
+
+Change directories to the folder containing this sample.
+
+Execute the following Maven command to build and run the app:
+
+```CMD
+mvn compile exec:java -Dexec.mainClass=example.demo.App
```

Comment:
I suspect this might be due to the particular details of the Maven packing, so it might not be worth trying to fix for this sample, but I noticed that when I execute the sample on the command line, before the program output itself, I get:

```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

Do you know if there's any easy way to fix or silence those messages so new users won't need to worry about them when they try running this sample?

Reply:
Good catch. I googled a [solution](https://stackoverflow.com/questions/7421612/slf4j-failed-to-load-class-org-slf4j-impl-staticloggerbinder) and pushed commit 2a43397 to resolve warnings.